router/boot.go

```diff
@@ -75,9 +84,10 @@ func main() {
 	// FIXME: have to launch cron first so generate-certs will generate the files nginx requires
 	go launchCron()
 
-	waitForInitialConfd(host+":"+etcdPort, timeout)
+	confdHost := strings.Split(etcdPeers[0], "://")[1]
```

Comment:
This doesn't work for some reason with my load balanced etcd quorum. I guess this is because I use a DNS name instead of an IP. I had to get rid of the `Split` and only keep the `etcdPeers[0]` in order to get it to work.